snuba/consumers/consumer_config.py

```diff
@@ -0,0 +1,218 @@
+from dataclasses import dataclass
```

Comment:
The contents of this file are essentially unchanged - just moved from the Rust consumer file so it can be used in other consumers